ow_lander/launch/spawn.launch

```diff
@@ -71,22 +56,42 @@
     -J j_grinder $(arg stowed_grinder_yaw)
     -unpause" />
 
+  <include file="$(find ow_lander)/launch/ros_controllers.launch"/>
+
+  <!-- Load the URDF, SRDF and other .yaml configuration files on the param server -->
+  <include file="$(find ow_lander)/launch/planning_context.launch">
+    <arg name="load_robot_description" value="false"/>

```

Comment:
load_robot_description is used as a conditional on the param that parses the xacro file, but "if" is not a valid attribute for <param>: http://wiki.ros.org/roslaunch/XML/param. That <param> line ends up overriding the one here in spawn.launch, which causes the lander to be disconnected from the frame tree and causes the 3 worlds to be dark. Remove <param> xacro line from planning_context.launch to fix all this. This load_robot_description arg will be obsolete and can also be removed.

Reply:
Oh I see, Great catch!

Reply:
Actually, according to http://wiki.ros.org/roslaunch/XML#if_and_unless_attributes it says that if/unless are supported on every tag and I did verify that the *param if* is a valid construct. However, I suspect that the value $(arg load_robot_description) is either getting a 'true' value or not being honored!

Reply:
Somehow it is getting a non-false value

Reply:
I think I have an intuition at what's happening. The only other file that is being loaded and uses the  same arg/param is move_group.launch and that one sets default value to true but I thought it wouldn't override the value being set by spawn.launch ... still looking at it but that's really weird

Reply:
Ok, so I have addressed it by setting **load_robot_description** to **false** when including `move_group.launch` within `spawn.launch`. I did verify that this doesn't break existing functionality.

Reply:
Note that this change also addresses the ticket https://babelfish.arc.nasa.gov/jira/browse/OCEANWATER-524 (which was created to ensure the configuration works for _atacama_y1a_, _europa_terminator_workspace_, initially only the _europa_terminator_ world could be loaded)